docs/intro.rst

```diff
@@ -47,13 +47,21 @@ Start by running the Schema Registry and the services it depends on: ZooKeeper a
     $ curl -X GET http://localhost:8081/subjects/Kafka-value/versions
       [1]
 
+    # Deletes all schema versions registered under the subject "Kafka-value"
+    $ curl -X DELETE http://localhost:8081/subjects/Kafka-value
```

Comment:
If we delete the entire subject here, the subsequent commands wouldn't work, i.e. if someone followed each of these commands, they wouldn't get the output that is shown. Perhaps move deletion to the end and then verify with a `GET` to fully demonstrate the functionality?